theano/tensor/tests/test_basic.py

```diff
@@ -1629,7 +1658,7 @@ def test_py_c_match():
     uint8=[numpy.arange(1, 256, dtype='uint8')],
     complex=(randc128_ranged(1, 1000, (2, 3)),),
     empty=(numpy.asarray([], dtype=config.floatX),),)
-_grad_broadcast_unary_arccosh = dict(normal=(rand_ranged(1, 1000, (2, 3)),),)
+_grad_broadcast_unary_arccosh = dict(normal=(rand_ranged(1 + _eps, 1000, (2, 3)),),)
```

Comment:
I don't think this one is useful, but it won't hurt the test quality. So I'll merge it.